backend/WebApi/Auth/JwtService.cs

```diff
@@ -77,8 +80,8 @@ private List<Claim> GetClaims(LfUser user)
         {
             new Claim(EmailClaimType, user.Email),
             new Claim(JwtRegisteredClaimNames.Sub, user.Id.ToString()),
-            //todo add project specific roles
             new Claim(RoleClaimType, user.Role.ToString()),
+            new Claim(ProjectsClaimType, JsonConvert.SerializeObject(user.Projects)),
```

Comment:
still need to swap this out for [JsonSerializer](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializer?view=net-7.0) from `System.Text.Json`

Reply:
Woops 🙂